lib/Service/RecentPagesService.php

```diff
@@ -104,9 +104,9 @@ public function forUser(IUser $user, int $limit = 10): array {
 				$pagePath = dirname($internalPath);
 			}
 
-			if ($pagePath === '.') {
-				$pagePath = '';
-			}
+			$pagePath = $pagePath === '.'
+				? ''
+				: str_replace(DIRECTORY_SEPARATOR, '-', $pagePath);
```

Comment:
In the page info model you are using `' - '` as a replacement:
https://github.com/nextcloud/collectives/blob/88719183ae673c80a9f3674a1645a7c15fb87e94/lib/Model/PageInfo.php#L139-L141

Is this distinction on purpose?

Reply:
Ah, good catch! Will fix it.